compiler/cpp/src/generate/t_java_generator.cc

```diff
@@ -2579,7 +2584,15 @@ void t_java_generator::generate_service_future_interface(t_service* tservice) {
   vector<t_function*> functions = tservice->get_functions();
   vector<t_function*>::iterator f_iter;
   for (f_iter = functions.begin(); f_iter != functions.end(); ++f_iter) {
-    indent(f_service_) << function_signature_future(*f_iter) << ";" << endl << endl;
+    t_function* t_f = *f_iter;
+    int mask;
```

Comment:
Question about this, I was going to inline it as "for (int mask ...)" but then I saw that f_iter is declared in a different line, and remembered about some compilers complaining about that inline definition, so I wasn't sure what the target compilers for thrift are. 100% safe to write the "for (int mask" idiom?